packages/app-runtime/test/runtime/Startup.test.ts

```diff
@@ -64,3 +64,46 @@ describe("Runtime Startup", function () {
         expect(selectedAccount.account.id.toString()).toBe(localAccount.id.toString());
     });
 });
+
+describe("Start Accounts", function () {
+    let runtime: AppRuntime;
+
+    let sessionA: LocalAccountSession;
+    let sessionB: LocalAccountSession;
+
+    beforeAll(async function () {
+        runtime = await TestUtil.createRuntime();
+        await runtime.start();
+
+        const accounts = await TestUtil.provideAccounts(runtime, 2);
+        sessionA = await runtime.selectAccount(accounts[0].id);
+        sessionB = await runtime.selectAccount(accounts[1].id);
+    });
+
+    afterAll(async () => await runtime.stop());
+
+    test("should not delete Account running startAccounts for an active Identity", async function () {
+        await runtime["startAccounts"]();
+        await expect(runtime.selectAccount(sessionA.account.id)).resolves.not.toThrow();
+    });
+
+    // TODO: Jest did not exit one second after the test run has completed.
```

Comment:
I'm a bit lost here. The reason for this must be in `await runtime["startAccounts"]();`. However in the following test, where the deletion job is executed, this doesn't occur. This was tested setting only for each test. The only difference in startAccounts between these tests it that here syncDatawallet is successful. @jkoenig134 Any idea?

Reply:
Hm, do we do a proper cleanup when removing the old accounts? Removing all references to controllers?

Reply:
I wouldn't vow for that. I only took what was in the previous deleteAccount method. Admittedly, I'm not too familiar with the wiring of Accounts. :D

Reply:
I think I did my best to clean up, but it's kind of hard to really get rid of everything..